modules/caddyhttp/fileserver/browsetplcontext.go

```diff
@@ -74,12 +75,18 @@ func (fsrv *FileServer) directoryListing(ctx context.Context, entries []fs.DirEn
 
 		size := info.Size()
 		fileIsSymlink := isSymlink(info)
+		symLinkRepresentation := ""
 		if fileIsSymlink {
 			path := caddyhttp.SanitizedPathJoin(root, path.Join(urlPath, info.Name()))
 			fileInfo, err := fs.Stat(fsrv.fileSystem, path)
 			if err == nil {
 				size = fileInfo.Size()
 			}
+
+			symLinkTarget, err := filepath.EvalSymlinks(path)
+			if err == nil {
+				symLinkRepresentation = name + " -> " + symLinkTarget
```

Comment:
Actually, how about if we use a Tabler SVG icon for this arrow?

```
<svg xmlns="http://www.w3.org/2000/svg" class="icon icon-tabler icon-tabler-arrow-narrow-right" width="24" height="24" viewBox="0 0 24 24" stroke-width="2" stroke="currentColor" fill="none" stroke-linecap="round" stroke-linejoin="round"><path stroke="none" d="M0 0h24v24H0z" fill="none"/><path d="M5 12l14 0" /><path d="M15 16l4 -4" /><path d="M15 8l4 4" /></svg>
```

Reply:
Here, do we just add the svg tag mentioned above to the `symLinkRepresentation` string or is there another/better way of doing it?

Reply:
Yeah, just replace the `" -> "` string with the `" <svg ...> "` string.

You could move that into a separate variable for readability though.

Reply:
I was just trying this and the `" <svg ...> "` gets displayed as it is inside the html. I believe the value passed to the go template needs to be of type template.HTML in order for it to parse it as html rather than the raw string. If so, how do you suggest we treat this?

Reply:
Ah, that's probably true. In that case, the correct way to handle this would be to create a new field in the `fileInfo` struct, maybe called `Symlink` or `SymlinkName` or `SymlinkPath` or something. Then the UI will render the arrow SVG icon and the symlink info. Does that make sense?

Reply:
@mholt I think I got what you mean. As you can see now, I added a field called `SymlinkPath` to the `fileInfo` struct. This new field is a slice of strings that can either be empty or containing two elements (first one being the name of the symlink and the second one being the target of the symlink). This slice gets filled in only when the file is a symlink and the `reveal_symlinks` directive is enabled. Therefore, in the `browse.html` file we can check if this slice is empty and in that case display the name otherwise display the first element of the slice followed by theS SVG icon followed by the second element of the slice. Is this a valid approach?